.github/workflows/tests.yml

```diff
@@ -53,6 +50,9 @@ jobs:
     name: ${{ matrix.os }}-py${{ matrix.python-version }}${{ matrix.LABEL }}
     runs-on: ${{ matrix.os }}
     timeout-minutes: 15
+    env:
+      MPLBACKEND: agg
+      PYTEST_ARGS: --pyargs kikuchipy --reruns 2 -n 2 --cov=kikuchipy
```

Comment:
Is there a specific test or reason to add in the reruns?  It might be better to just add the `@pytest.mark.flaky(reruns=2)` to the tests which are flaky and might fail?

Reply:
Yes, the reason was a flaky test on a macOS run ([here](https://github.com/pyxem/kikuchipy/actions/runs/6722762882/job/18271402048)). The test check availability of datasets on Zenodo. I've experienced some flaky-ness in the past, so I figured this change was overdue.